### PR TITLE
[Admin] Show the total price of a product in the product folder

### PR DESCRIPTION
**Pull request by MickhaelSh**
## Description
Added cell to product table, fixed warnings in product form

#### Screenshots

Знімок екрана 2022-11-28 161056

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅ All tests passed locally and linter has no warnings and errors
- [x] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/product-materials-container/tests/product-info-container.spec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+describe('CartHeader', () => {
+  it('should consist 1 item in <Badge />', () => {
+    render(<ProductMaterialsContainer {...mockProps} />);
+
+    const mainColor = screen.getByText(mockProps.mainColors[0].name[0].value);
+
+    expect(mainColor).toBeInTheDocument();
+  });
+  it('should consist 1 item in <Badge />', () => {
+    render(<ProductMaterialsContainer {...mockProps} />);
```
Comment (tarasSanotskyi) [inline src/containers/product-materials-container/tests/product-info-container.spec.js:34]: render in every test?
use beforeEach

File: src/containers/product-species-container/tests/product-species-container.spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+describe('CartHeader', () => {
+  it('should consist 1 item in <Badge />', () => {
+    render(<ProductSpeciesContainerconst {...mockProps} />);
+
+    const model = screen.getByText(mockProps.models[0].name[0].value);
+
+    expect(model).toBeInTheDocument();
+  });
+  it('should consist 1 item in <Badge />', () => {
+    render(<ProductSpeciesContainerconst {...mockProps} />);
```
Comment (tarasSanotskyi) [inline src/containers/product-species-container/tests/product-species-container.spec.js:35]: same

File: src/containers/product-materials-container/tests/product-info-container.spec.js (hunk 1 above)
Comment (MickhaelSh) [reply src/containers/product-materials-container/tests/product-info-container.spec.js]: fixed

File: src/containers/product-species-container/tests/product-species-container.spec.js (hunk 2 above)
Comment (MickhaelSh) [reply src/containers/product-species-container/tests/product-species-container.spec.js]: fixed

**Review by Pipuss — DISMISSED** [review]

**Review by tarasSanotskyi — DISMISSED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by Tolik170 — DISMISSED** [review]

File: src/containers/images-preview-container/images-preview-container.js [hunk 3]
```diff
@@ -111,7 +104,7 @@ const ImagesPreviewContainer = ({ src, labels, multiple, imageHandler }) => {
               onChange={handleChange}
               inputProps={{ 'aria-label': 'controlled' }}
               disabled={file.src === preview}
-              name={file.index}
+              name={String(file.index)}
```
Comment (byavdoshniak) [inline src/containers/images-preview-container/images-preview-container.js:107]: check if it's needed and remove if no

**Review by byavdoshniak — DISMISSED** [review]

File: src/containers/images-preview-container/images-preview-container.js (hunk 3 above)
Comment (MickhaelSh) [reply src/containers/images-preview-container/images-preview-container.js]: fixed

**Review by tarasSanotskyi — APPROVED** [review]

**Review by Pipuss — APPROVED** [review]

**Pull request merged**